src/Nethermind/Nethermind.JsonRpc/Modules/Eth/FeeHistory/FeeHistoryOracle.cs

```diff
@@ -109,13 +109,16 @@ private readonly record struct BlockFeeHistorySearchInfo(
             BlockFeeHistorySearchInfo BlockFeeHistorySearchInfoFromBlock(Block b)
             {
                 BlobGasCalculator.TryCalculateFeePerBlobGas(b.Header, out UInt256 feePerBlobGas);
+
+                var maxBlobGasPerBlock = (double)(block.TargetBlobCount * 2 ?? Eip4844Constants.GetMaxBlobsPerBlock()) * Eip4844Constants.GasPerBlob;
```

Comment:
We have `MaxBlobCount` in `PayloadAttributes` so why `TargetBlobCount * 2`?

Reply:
It shouldn't be a part of header? Maybe it is something to be discussed broader with other teams.

Reply:
https://github.com/ethereum/consensus-specs/pull/4042
this PR solves the issues, but this will not be a part of devnet-5

Reply:
Ok so:
1. We have MaxBlobCount in this PR in block header
2. It is not in original EIP, but it is in next PR, but not for devnet ?

Reply:
we dont have MaxBlobCount in the header in this PR

Reply:
maybe we just use `Eip4844Constants.MaxBlobGasPerBlock` instead of `TargetBlobCount * 2`?

Reply:
yes, that would also work. I checked reth's implementation and they are also doing the same

Reply:
reverted to `TargetBlobCount * 2`